File: proyecto_java/proyecto_final/src/paquete04/problemaBasquetball.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package paquete04;

import java.util.Scanner;

/**
 *
 * @author utpl
 */
public class problemaBasquetball {

    public static String obtenerDatosBasquetball() {

        String cadena;

        Scanner entrada = new Scanner(System.in);
        System.out.println("Ingrese nombre del participante");
        String nombrePersona = entrada.nextLine();

        System.out.println("Ingrese su edad");
        int edad = entrada.nextInt();

        System.out.println("Ingrese su posicion preferida de juego");
        entrada.nextLine();
        String posicion = entrada.nextLine();

        System.out.println("Ingrese su estatura");
        double estatura = entrada.nextDouble();

        System.out.println("Ingrese su ciudad de residencia");
        entrada.nextLine();
        String ciudad = entrada.nextLine();

        cadena = String.format("\nLos datos ingresados son: "
                + "\nNombre: %s\nEdad: %d\nCiudad: %s\nEstatura: %.2f\n"
                + "Posicion: %s\n",
                nombrePersona, edad, ciudad, estatura, posicion);

        return cadena;

    }
}

File: proyecto_python/proyectofinal2b/problemaBasquetball.py
def obtenerDatosBasquetball():
    print("Ingrese el nombre del participante:")
    nombre = input()

    print("Ingrese la edad del participante:")
    edad = input()
    edad = float(edad)

    print("Ingrese su posicion preferida de juego:")
    posicion = input()

    print("Ingrese su ciudad de residencia")
    ciudad = input()

    print("Ingrese su estatura")
    estatura = input()

    cadena = (f"\nLos datos ingresados son: "
              f"\nNombre: {nombre}\nEdad: {edad}\nCiudad: {ciudad}"
              f"\nPosicion preferida: {posicion}\nEstatura: {estatura}\n")

    return cadena